java/pom.xml

```diff
@@ -47,7 +47,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
-        <protobuf-version>3.12.2</protobuf-version>
+        <protobuf-version>3.21.4</protobuf-version>
```

Comment:
Shall we go ahead with `3.19.4` to be consistent with `stachschema-sdks`? Do we need to check how the deserialization is working for STACH v1 and v2 responses? @Ramanth

Reply:
If I may, the vulnerability CVE-2022-25647 is present in 3.19.4 too. The same vulnerability is mentioned here:
https://sbom.lift.sonatype.com/report/T1-a0368c8f29fdaa555824-2f4ed5e8469de1-1658413948-54b685bd0fb94efb9ddff83db59a5dc3